arviz/plots/backends/matplotlib/kdeplot.py

```diff
@@ -144,11 +144,13 @@ def plot_kde(
         else:
             fill_kwargs.setdefault("alpha", 0)
             if fill_kwargs.get("alpha") == 0:
-                ax.plot(x, density, label=label, **plot_kwargs)
+                label = plot_kwargs.setdefault("label", label)
+                ax.plot(x, density, **plot_kwargs)
                 fill_func(fill_x, fill_y, **fill_kwargs)
             else:
+                label = fill_kwargs.setdefault("label", label)
```

Comment:
I think setdefault doesn't return anything, so it's output should not be stored anywhere.

Reply:
I referred [this](https://docs.python.org/3/library/stdtypes.html#dict.setdefault) link. I have used it stored its value because `label` is used in line 154.

Reply:
true, good catch